### PR TITLE
fix: add capacity error message

### PR DESCRIPTION
**Pull request by debjyoti-pandit**
fixes: https://issues.redhat.com/browse/OCSBZM-6225

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
/approve cancel
@bipuladh in case u have other thoughts regarding this BZ... LGTM...

**Comment by SanjalKatiyar** [comment]
update the PR title correctly as per the guidelines...

**Comment by SanjalKatiyar** [comment]
> update the PR title correctly as per the guidelines...

/lgtm cancel

**Comment by SanjalKatiyar** [comment]
> /approve cancel @bipuladh in case u have other thoughts regarding this BZ... LGTM...

/lgtm

File: packages/odf/modals/add-capacity/add-capacity-modal.tsx [hunk 1]
```diff
@@ -434,7 +434,7 @@ export const AddCapacityModal: React.FC<AddCapacityModalProps> = ({
           ))}
         {errorMessage && (
           <Alert isInline variant="danger" title={t('An error occurred')}>
-            {(errorMessage as any)?.message}
+            {(errorMessage as any)?.message || errorMessage}
```
Comment (bipuladh) [inline packages/odf/modals/add-capacity/add-capacity-modal.tsx:437]: why are both required?

Comment (debjyoti-pandit) [reply packages/odf/modals/add-capacity/add-capacity-modal.tsx:437]: from this function, we are returning single [string](https://github.com/red-hat-storage/odf-console/blob/7a07e8e7ba0f8091416efd7783e719870eabab84/packages/odf/modals/add-capacity/add-capacity-modal.tsx#L290)

**Pull request merged**